spec/features/preassembly_run/media_video_spec.rb

```diff
@@ -11,58 +11,12 @@
   let(:bundle_dir) { Rails.root.join('spec/test_data/media_video_test') }
   let(:bare_druid) { 'vd000bj0000' }
   let(:object_staging_dir) { Rails.root.join(Settings.assembly_staging_dir, 'vd', '000', 'bj', '0000', bare_druid) }
-  let(:cocina_model_world_access) { instance_double(Cocina::Models::Access, view: 'world') }
-  let(:item) { instance_double(Cocina::Models::DRO, type: Cocina::Models::ObjectType.media, access: cocina_model_world_access) }
-  let(:dsc_object_version) { instance_double(Dor::Services::Client::ObjectVersion, openable?: true) }
-  let(:dsc_object) { instance_double(Dor::Services::Client::Object, version: dsc_object_version, find: item) }
-  let(:exp_content_md) do
-    <<~XML
-      <contentMetadata objectId="vd000bj0000" type="media">
-        <resource sequence="1" id="vd000bj0000_1" type="video">
-          <label>Video file 1</label>
-          <file id="vd000bj0000_video_1.mp4" preserve="yes" publish="yes" shelve="yes">
-            <checksum type="md5">ee4e90be549c5614ac6282a5b80a506b</checksum>
-          </file>
-          <file id="vd000bj0000_video_1.mpeg" preserve="yes" publish="no" shelve="no">
-            <checksum type="md5">bed85c6ffc2f8070599a7fb682852f30</checksum>
-          </file>
-          <file id="vd000bj0000_video_1_thumb.jp2" preserve="yes" publish="yes" shelve="yes">
-            <checksum type="md5">4b0e92aec76da9ac98567b8e6848e922</checksum>
```

Comment:
maybe I'm missing it, but I'm not seeing this thorough a comparison for the contents of the file;  media passes us checksums in their file structure, I believe.  Ditto for various other tests with checksums provided.

Reply:
Need to ensure precomputed checksums from video_1_thumb.jp2.md5 are checked in the PreAssembly::FromManifest::StructuralBuilder)

Reply:
so this is tested now ... how?  It's not surfaced in the test

Reply:
Testing this stuff has moved to the StructuralBuilder tests.